kwok/getuidmap.go

```diff
@@ -10,10 +10,6 @@ const (
 	contextName      = "kwok"
 	clusterServerURL = "http://127.0.0.1:8080"
 	scriptPath       = "../scripts/get-uid-map.sh"
-
-	FailedToSetClusterMsg = "failed to set cluster: %w"
-	FailedToSetContextMsg = "failed to set context: %w"
```

Comment:
Are those necessary for this change?

Reply:
Nope. but we can remove it as part of this. or should I create a separate PR for this?

Reply:
If it is totally not related I'd separate it